src/raft.rs

```diff
@@ -1476,16 +1476,18 @@ impl<T: Storage> Raft<T> {
         if let Some(to_send) = more_to_send {
             self.send(to_send)
         }
+
+        return Ok(());
```

Comment:
Should avoid using return in the last line.

Reply:
updated,  also other func